### ISSUE TITLE
Use Attention to False: dimension fix

### ISSUE THREAD

**Issue by giakou4** [mskcc/mimsi#26]
mimsi/model/mi_msi_model.py  line 164 correct code: ` S = torch.mean(I, 0)`

**Comment by john-ziegler** [comment]
Hi Nikos - what's the issue that you're seeing here? Line 164 is ` A = nn.functional.softmax(A, dim=1)`, and the mean function that you have shown is already present [here](https://github.com/mskcc/mimsi/blob/master/model/mi_msi_model.py#L169) at line 169. Perhaps there is a versioning issue?

**Comment by giakou4** [comment]
No problem, I confused lines
